cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -38,6 +39,34 @@ message OptionalMap {
     map<string, string> data = 1;
 }
 
+// File or dataset or cube derived from a single SEG-Y file
+message CloudStorageFile {
+    string uuid = 1;
+    int64 id = 2;
+    ExternalId external_id = 3;
+    string name = 4;
+    map<string, string> metadata = 5;
+    HeaderOffsets header_offsets = 6;
+}
+
+// Metadata related to interpreting SEG-Y files.
+message HeaderOffsets {
```

Comment:
To be pedandic, the `source_group_scalar_override` is not a header offset. What about `SegyOverrides`?

Reply:
👍